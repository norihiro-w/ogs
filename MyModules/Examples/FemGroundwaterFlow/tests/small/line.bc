GINA - Boundary Condition
#BOUNDARY_CONDITION
 $PCS_TYPE
  LIQUID_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POINT POINT1
 $DIS_TYPE
  CONSTANT   100000
#STOP
