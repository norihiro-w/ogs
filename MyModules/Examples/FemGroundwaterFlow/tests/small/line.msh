#FEM_MSH				
$PCS_TYPE				
HEAT_TRANSPORT				
$NODES				
11
0	0	0	0	
1	10	0	0	
2	20	0	0	
3	30	0	0	
4	40	0	0	
5	50	0	0	
6	60	0	0	
7	70	0	0	
8	80	0	0	
9	90	0	0	
10	100	0	0	
$ELEMENTS				
10
0	0	line	0	1
1	0	line	1	2
2	0	line	2	3
3	0	line	3	4
4	0	line	4	5
5	0	line	5	6
6	0	line	6	7
7	0	line	7	8
8	0	line	8	9
9	0	line	9	10
#STOP				
