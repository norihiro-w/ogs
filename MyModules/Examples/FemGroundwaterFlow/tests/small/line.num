#NUMERICS
 $PCS_TYPE
  LIQUID_FLOW
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      2 1.e-016       1000           1.0   1       4
; $NON_LINEAR_ITERATION
; ; method error_type max_iterations relaxation tolerance 
;  NEWTON LMAX  2 0.0 1e-3
;  LINEAR 1e-3            25             0.0
 $NON_LINEAR_SOLVER  
 ; method error_tolerance local_tol  max_iterations relaxation
;  NEWTON 1e-15 0 25 0.0
 ; method error_tolerance max_iterations relaxation
;  LINEAR 1e-3            25             0.0
  SNES 1e-3            2             0.0
#NUMERICS
 $PCS_TYPE
  HEAT_TRANSPORT
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      0 1.e-012       1000           1.0   1       4
 $ELE_GAUSS_POINTS
  2
 $NON_LINEAR_SOLVER  
 ; method error_tolerance max_iterations relaxation
  LINEAR 1e-3            25             0.0
;  PICARD 1e-3            25             0.0
#STOP

