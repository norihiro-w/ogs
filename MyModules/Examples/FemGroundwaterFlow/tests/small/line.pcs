GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
  LIQUID_FLOW
#STOP
