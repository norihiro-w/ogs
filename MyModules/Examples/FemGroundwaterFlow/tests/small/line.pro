<?xml version="1.0"?>
<ogs6>
<coupling>
	<P algorithm="Serial" convergence="FemFunctionConvergenceCheck" max_itr="1" epsilon="1e-4">
		<out>PRESSURE1</out>
		<out>VELOCITY</out>
		<out>TEMPERATURE1</out>
		<problems>
			<M type="LIQUID_FLOW">
				<out>PRESSURE1</out>
			</M>
			<M type="PRESSURE_TO_ELEMENT_VELOCITY">
				<in>PRESSURE1</in>
				<out>VELOCITY</out>
			</M>
			<M type="HEAT_TRANSPORT">
				<in>VELOCITY</in>
				<out>TEMPERATURE1</out>
			</M>
		</problems>
	</P>
</coupling>
</ogs6>

