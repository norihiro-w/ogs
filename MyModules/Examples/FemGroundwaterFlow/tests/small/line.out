GeoSys-OUT: Output ------------------------------------------------
#STOP

#OUTPUT
 $NOD_VALUES
  PRESSURE1
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

